python/ray/serve/tests/test_standalone2.py

```diff
@@ -138,6 +138,49 @@ def f(*args):
         serve.shutdown()
 
 
+def test_updating_status_message(shutdown_ray):
+    ray.init(num_cpus=2)
+    client = serve.start(detached=True)
+
+    @serve.deployment(
+        num_replicas=5,
+        ray_actor_options={"num_cpus": 1},
+    )
+    def f(*args):
+        pass
+
+    original_slow_startup_warning_period_s = (
+        client._controller._get_slow_startup_warning_period_s.remote()
+    )
+    original_slow_startup_warning_s = (
+        client._controller._get_slow_startup_warning_s.remote()
+    )
+    # Lower slow startup warning threshold to 1 second to reduce test duration
+    client._controller._set_slow_startup_warning_period_s.remote(1)
+    client._controller._set_slow_startup_warning_s.remote(1)
+    f.deploy(_blocking=False)
+
+    def updating_message():
+        deployment_status = client.get_serve_status().deployment_statuses[0]
+        message_substring = "more than 1s to be scheduled."
+        return (deployment_status.status == "UPDATING") and (
+            message_substring in deployment_status.message
+        )
+
+    wait_for_condition(updating_message, timeout=2)
+    # Reset slow startup warning threshold in case bugs that cause different
```

Comment:
Nit– can we rephrase this comment to "Reset slow startup warning threshold to prevent state sharing across unit tests."